tests/test.hs

```diff
@@ -443,19 +443,16 @@ testGroupsWithLibs name (DependentTests libTests nonlibTests) =
 
 -- | Creates a [TestTree] that runs without parallelism
 testSequentially :: String -> SequentialTests -> [TestTree]
-testSequentially name (SequentialTests indexedTests) =
-  let grouped = (\(t, n) -> (testGroup (mkName n) [t], n)) <$> indexedTests
-      -- turns i.e. [a, b, c, d] into [(a, b), (b, c), (c, d)]
-      pairs = zip <*> tail $ grouped
+testSequentially name (SequentialTests tests) =
+  let grouped = (\(t, n) -> (testGroup (mkName n) [t], n)) <$> (zip tests [0 :: Int ..])
```

Comment:
The need to create a `testGroup` is that there is no way to get the name of the given `TestTree`, right?
If that is the case, a comment informing this could be helpful if someone wants to change it later on.

Reply:
This is correct, and a comment has been added.